### DISCUSSION TITLE
Problems with integration

### DISCUSSION THREAD

**Discussion by aalllop** [llamafilm/tesla-http-proxy-addon#114] [General]
I have installed tesla http proxy and configured tesla integration to use it. Since today I can´t made orders to the car. I still can view sensors information but not change charging amps, close doors, etc from the integration. 

I tried to regenerate the token. The process ends correctly and I can register one more time Tesla Integration on HA. Are anyone with the same problem? What could be happening?

**Comment by llamafilm** [comment]
If you can read sensors but can’t send commands, then either you’re not actually using Fleet API, or perhaps there’s something wrong with your public key.
